features/update/async_accepted/feature.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+from datetime import timedelta
+
+from temporalio import activity, workflow
+from temporalio.client import RPCError, WorkflowHandle, WorkflowUpdateFailedError
+from temporalio.exceptions import ApplicationError
+
+from harness.python.feature import Runner, register_feature
+
+
+@workflow.defn
+class Workflow:
+    am_done = False
+
+    @workflow.run
+    async def run(self) -> str:
+        await workflow.wait_condition(lambda: self.am_done)
+        return "Hello, World!"
+
+    @workflow.update
+    async def do_maybe_sleepy_update(self, sleep: bool) -> int:
+        if sleep:
+            await asyncio.sleep(2)
```

Comment:
For my .NET ones, I had this block on signal so that we didn't have to sleep in a workflow in our tests (sleep causes flakes and in general slows the system down)

Reply:
Sure, totally reasonable, can change them to signals.